Tools/Scripts/webkitpy/w3c/test_exporter.py

```diff
@@ -240,13 +240,28 @@ def create_branch_with_patch(self, patch):
             self._run_wpt_git(['checkout', '-b', self._branch_name])
 
         try:
-            self._run_wpt_git(['apply', '--index', patch, '-3'])
+            output = self._run_wpt_git(['apply', '--index', patch, '-3'], capture_output=True)
+            if output.returncode:
+                _log.error(f'Failed to apply patch!')
+                _log.error(f"{output.stdout.decode('utf-8')}\n{output.stderr.decode('utf-8')}")
```

Comment:
```suggestion
            output = self._run_wpt_git(['apply', '--index', patch, '-3'], stderr=subprocess.STDOUT, capture_output=True)
            if output.returncode:
                _log.error(f'Failed to apply patch!')
                _log.error(f"{output.stdout.decode('utf-8', 'replace')}")
```

This:

 * requires importing `subprocess`,
 * handles the merging of `stdout` and `stderr` at a lower level so they appear intermixed as they would in a terminal, and
 * handles the output not being valid UTF-8 (which can happen with some patches I think?).